### DISCUSSION TITLE
How you scale? microservice

### DISCUSSION THREAD

**Discussion by AnymGameMaster** [remix-run/remix#5919] [Q&A]
Hello,

On microservices you have different Servers for Each api like /users /Posts 

But how you apply it in remix? Backend and frontend is in one Code so when I Make a call to /Users or anything Else then it goes all in the same server 

how can I add microservice to remix

**Comment by sergiodxa** [comment]
You could create your micro services as usual and fetch them from your Remix loaders and actions.
